Procfile.dev

```diff
@@ -1,2 +1,2 @@
 web: rails s -p 3000
-client: sh -c 'rm app/assets/webpack/* || true && cd client && bundle exec rake react_on_rails:locale && npm run build:development'
+client: sh -c 'rm -rf public/webpack/development/* || true && cd client && bundle exec rake react_on_rails:locale && yarn run build:development'
```

Comment:
This file was generated so I just left it alone. I'm fine with updating this to npm.